packages/api/tests/endpoints/test-granules.js

```diff
@@ -1726,13 +1730,16 @@ test.serial('create (POST) creates new granule with associated execution in dyna
       collection_cumulus_id: t.context.collectionCumulusId,
     }
   );
-
+  const fetchedESRecord = await t.context.esGranulesClient.get(
```

Comment:
Need to update the test title here as well

Reply:
[`9e7f401` (#2589)](https://github.com/nasa/cumulus/pull/2589/commits/9e7f401d7f28b92ea71722f414f9a290fa430faf)